composer.json

```diff
@@ -7,16 +7,22 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=7.4",
-        "laravel/framework": "^5.6|^6.0|^7.0|^8.0",
+        "php": ">=7.4|^8.0",
+        "laravel/framework": "^5.6|^6.0|^7.0|^8.0|^9.0",
         "laravel/nova": "^2.0|^3.0",
-        "spatie/laravel-medialibrary": "^8.0|^9.0"
+        "spatie/laravel-medialibrary": "^8.0|^9.0|^10.0"
     },
     "autoload": {
         "psr-4": {
             "Ebess\\AdvancedNovaMediaLibrary\\": "src/"
         }
     },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://nova.laravel.com"
+        }
+    ],
```

Comment:
@nasrulhazim why is this needed? Can you remove this please?

Reply:
Ah.. i should remove that. accidentally added.